core/src/main/java/com/sequenceiq/cloudbreak/controller/validation/template/TemplateValidatorAndUpdater.java

```diff
@@ -236,7 +239,7 @@ private void validateVolumeCount(VolumeTemplate value, VmType vmType, VolumePara
                 if (value.getVolumeCount() > config.maximumNumber()) {
                     validationBuilder.error(String.format("Max allowed volume count for '%s': %s", vmType.value(), config.maximumNumber()));
                 } else if (!(isIDBrokerInstanceGroup(instanceGroup) || isComputeInstanceGroup(instanceGroup)
-                        || isCoordinatorAndExecutorInstanceGroup(bluePrintText)) &&
+                        || isCoordinatorAndExecutorInstanceGroup(bluePrintText) || isSdxComputeInstanceHostGroup(instanceGroup)) &&
```

Comment:
Another dumb request, move this check to before `isCoordinatorAndExecutorInstanceGroup` so all the `instanceGroup` checks can be together.